package.yaml

```diff
@@ -1,5 +1,5 @@
 name: bugsnag-haskell
-version: 0.0.3.1
+version: 0.0.4
```

Comment:
Let's not bump version in PR. I'll do that later, possibly batching together more changes.

Reply:
```suggestion
version: 0.0.3.1
```